builder/crawler/lib.go

```diff
@@ -206,24 +206,30 @@ func GetAdjuntosCedula(url string, ficha *shared.Ficha, actuacion *shared.Actuac
 		}).Warn("Failed to decode json")
 		return nil, err
 	}
-	documentos := make([]*shared.Documento, len(adjuntos))
-	for i, adjunto := range adjuntos {
+	documentos := make([]*shared.Documento, 0, len(adjuntos))
+	for _, adjunto := range adjuntos {
+		if val, found := adjunto["adjuntoId"]; !found || val == nil {
+			log.WithFields(log.Fields{
+				"adjunto": adjunto,
+			}).Warn("skipping adjunto")
+			continue
+		}
 		url := fmt.Sprintf("https://eje.juscaba.gob.ar/iol-api/api/public/expedientes/cedulas/adjuntoPdf?filter=%%7B%%22aacId%%22:%v,%%22expId%%22:%v,%%22ministerios%%22:false%%7D",
-			adjunto["adjuntoId"],
+			int(adjunto["adjuntoId"].(float64)),
 			ficha.ExpId,
 		)
-		documentos[i] = &shared.Documento{
+		documentos = append(documentos, &shared.Documento{
 			URL:                url,
```

Comment:
Should go to another commit ?

Reply:
it is related to the continue that's a consequence of the float64 change

Reply:
(casting nil to float64 is a fatal error)